src/jimgw/single_event/detector.py

```diff
@@ -345,30 +342,29 @@ def fd_response(
                 - ra (Float): Right ascension in radians
                 - dec (Float): Declination in radians
                 - psi (Float): Polarization angle in radians
-                - gmst (Float): Greenwich mean sidereal time in radians
-                - t_c (Float): Difference between geocent time and trigger time in sec
-            trigger_time (Float): Trigger time of the data in seconds.
+                - geocent_time (float): The geocentric time in sec
             **kwargs: Additional keyword arguments.
 
         Returns:
             Array: Complex strain measured by the detector in frequency domain, obtained by
                   combining the antenna patterns for each polarization mode.
         """
-        ra, dec, psi, gmst = params["ra"], params["dec"], params["psi"], params["gmst"]
+        ra, dec, psi = params["ra"], params["dec"], params["psi"]
+        gmst = compute_gmst(params["geocent_time"])
```

Comment:
Per the discussion, let's keep the use of `compute_gmst` out of the detector response for now. Please revert this back to using a static `gmst`